src/main/java/codeu/model/data/Message.java

```diff
@@ -66,13 +69,28 @@ public String getContent() {
     return content;
   }
 
+ /** Returns mentioned user. */
+  public String getMentionedUser(){
+    return mentionedUser;
+  }
+
   public String getStyledContent(String s) {
     Parser parser = Parser.builder().build();
     Node document = parser.parse(s);
     HtmlRenderer renderer = HtmlRenderer.builder().build();
     return renderer.render(document);
   }
 
+  public String getStyledContentandMentioned(String s){
+    String styled = getStyledContent(s); 
+    for (int i = 0; i < styled.length(); i++){
+      if ((styled.substring(i, i + getMentionedUser().length())).equals(getMentionedUser()))
+        styled = styled + " Menitoned User Found";
```

Comment:
I think you'll be able to delete this part and just use the regex thing I said above to print the list of mentioned users